docs/source/app_developers_guide/go_sdk_import.rst

```diff
@@ -0,0 +1,59 @@
+********************
+Importing the Go SDK
+********************
+
+.. note::
+   The Sawtooth Go SDK assumes that you have the latest version of Go,
+   more information on installing and configuring can be found at `go
```

Comment:
The Sawtooth documentation convention is to either show the URL or use more descriptive text for the link. For example:

"... can be found at https://github.com/golang/go."

or 

"... can be found in the \`golang/go repository <https://github.com/golang/go>\`__."

In the second example, please note the two underscores after the URL target - this prevents a doc build error in case the documentation has another reference to the same URL.